packages/xo-web/src/common/xo/index.js

```diff
@@ -345,9 +345,8 @@ export const subscribeRoles = createSubscription(
 
 export const subscribeIpPools = createSubscription(() => _call('ipPool.getAll'))
 
-export const subscribeResourceCatalog = createSubscription(() =>
-  _call('cloud.getResourceCatalog')
-)
+export const subscribeResourceCatalog = ({ filters } = {}) =>
```

Comment:
This is not a `subscribe`, this is a `createSubscription`.

Reply:
I've followed what have been done, it was always like this !
https://github.com/vatesfr/xen-orchestra/blob/cd378f016855187b5a042c7399d33ff6f4a78942/packages/xo-web/src/common/xo/index.js#L332-L340